django_loci/admin.py

```diff
@@ -40,5 +42,15 @@ class ObjectLocationInline(AbstractObjectLocationInline):
     form = ObjectLocationForm
 
 
+_admin_site_get_urls = admin.site.get_urls
+
+
+def get_urls():
```

Comment:
It is the most clean that I found to add these views. I if add these views to https://github.com/openwisp/django-loci/blob/master/django_loci/base/admin.py#L60 they will look like these `admin/django_loci/location/<my-part>` and that won't work because first parameter after `location/` is expected to be `pk`

Reply:
I'm not sure of this, if we don't do the override in the base files we will have to repeat this code in openwisp-controller, it looks to me that you have to try harder.

Have you tried adding:

```python
url(r'^geocode/$', admin.site.admin_view(geocode_view)),
url(r'^reverse_geocode/$', admin.site.admin_view(reverse_geocode_view))
```

to this list? https://github.com/openwisp/django-loci/blob/master/django_loci/base/admin.py#L65

Reply:
PS: use `reverse-geocode` instead of 'reverse_geocode` for the URL path, for consistency with the other OpenWISP modules in which we generally use dashes instead of underscores